client/js/components/SearchBar/styles.js

```diff
@@ -0,0 +1,33 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+
+const styles = StyleSheet.create({
+  searchBar: {
+    backgroundColor: 'rgb(227,227,227)',
+    borderRadius: 9,
+    width: 245,
+    height: 24,
+    justifyContent: 'space-between',
+  },
+  textField: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height: 20,
+    alignSelf: 'flex-start',
+    marginTop: 30,
+  },
+  searchBarContainer: {
+    paddingTop: 100,
```

Comment:
We should not be applying this much padding. Use flexbox properties to style your screen.

Reply:
No heading component yet so the padding is there so we can see the search bar I was working on, or else it will hidden under the notch.